server/public/reset.css

```diff
@@ -3,5 +3,21 @@
 }
 
 body {
+  background: #fafafa;
+  display: flex;
+  justify-content: center;
+  min-height: 100vh;
   margin: 0;
 }
+
+.bg-background {
+  background-color: #fafafa !important;
+}
+
+.text-primary {
+  color: #0195f6 !important;
+}
+
+.btn-primary {
+  background-color: #0195f6 !important;
+}
```

Comment:
If you are overriding CSS from a library, you should put those styles in a separate `overrides.css` file.